src/test/burn/TestData/WixStdBaTests/BundleA_v10/BundleA_v10.wixproj

```diff
@@ -0,0 +1,20 @@
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
+<Project Sdk="WixToolset.Sdk">
+  <PropertyGroup>
+    <OutputType>Bundle</OutputType>
+    <BA>hyperlinkLicense</BA>
+    <UpgradeCode>{7D977157-06C9-4176-A931-AC16E18AAB51}</UpgradeCode>
+    <DefineConstants>$(DefineConstants);Version=1.0</DefineConstants>
+    <OutputName>WixStdBaTest1_v10</OutputName>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\..\Templates\Bundle.wxs" Link="Bundle.wxs" />
+    <Compile Include="..\BundleA\BundleA.wxs" Link="BundleA.wxs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\PackageA\PackageA.wixproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="WixToolset.Bal.wixext" />
+  </ItemGroup>
+</Project>
```

Comment:
@barnson I added this bundle to test wixstdba downgrade behavior when downgrade failures aren't suppressed. If you install BundleA_v11 or BundleA_v12 first, and then run BundleA_v10 then BundleAv_10 hangs. In full UI, it shows all non-page controls but never shows any page so all you can do is cancel. With no UI, it hangs. I don't know if you want me to create a new issue or if you'll fix that as part of https://github.com/wixtoolset/issues/issues/6677.

Reply:
I'll take a look.